bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/bridge/json/JsonBridgeAPI.java

```diff
@@ -83,19 +83,19 @@
 class JsonBridgeAPI implements BridgeAPI {
     private final Logger logger = LoggerFactory.getLogger(JsonBridgeAPI.class);
 
-    private static final GetDeviceStatus GETDEVICESTATUS = new JCgetDeviceStatus();
-    private static final GetFirmware GETFIRMWARE = new JCgetFirmware();
-    private static final GetLANConfig GETLANCONFIG = new JCgetLANConfig();
-    private static final GetProducts GETPRODUCTS = new JCgetProducts();
-    private static final GetScenes GETSCENES = new JCgetScenes();
-    private static final GetWLANConfig GETWLANCONFIG = new JCgetWLANConfig();
-    private static final Login LOGIN = new JClogin();
-    private static final Logout LOGOUT = new JClogout();
-    private static final RunProductDiscovery RUNPRODUCTDISCOVERY = new JCrunProductDiscovery();
-    private static final RunProductIdentification RUNPRODUCTIDENTIFICATION = new JCrunProductIdentification();
-    private static final RunProductSearch RUNPRODUCTSEARCH = new JCrunProductSearch();
-    private static final RunScene RUNSCENE = new JCrunScene();
-    private static final SetSceneVelocity SETSCENEVELOCITY = new JCsetSceneVelocity();
+    private final GetDeviceStatus GETDEVICESTATUS = new JCgetDeviceStatus();
```

Comment:
You should probably change the casing If these variables are not static final anymore. They should not be uppercase.

Reply:
Good point. Although I would like to hear Guenther’s comments on the fundamentals of the change from using ‘static’ before I address the code styling.

Reply:
Ok. I changed the capitalisation.